templates/default/haproxy.cfg.erb

```diff
@@ -95,7 +95,6 @@ frontend <%= frontend %>
 <% end -%>
 <% unless f['use_backend'].nil? %>
 <% f['use_backend'].flatten.uniq.each do | backend |%>
-<% backend.each do | backend | %>
   use_backend <%= backend %>
 <% end -%>
```

Comment:
I already removed that one in the change before, there are 2 ends, one for the do, and one for the unless